viewer/src/utilities/Spinner.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Cognite AS
+ */
+
+import css from './spinnerStyles.css';
```

Comment:
We should make sure that the user can override the style if they want to :)

Reply:
I think there is nothing impossible right now. Just specifying a more specific CSS selector should do the trick.